producer/src/main/kotlin/streams/serialization/JacksonUtil.kt

```diff
@@ -2,28 +2,33 @@ package streams.serialization
 
 import org.codehaus.jackson.JsonGenerator
 import org.codehaus.jackson.JsonProcessingException
-import org.codehaus.jackson.map.ObjectMapper
 import org.codehaus.jackson.Version
 import org.codehaus.jackson.map.JsonSerializer
+import org.codehaus.jackson.map.ObjectMapper
 import org.codehaus.jackson.map.SerializationConfig
 import org.codehaus.jackson.map.SerializerProvider
 import org.codehaus.jackson.map.module.SimpleModule
-import org.neo4j.graphdb.spatial.CRS
-import org.neo4j.graphdb.spatial.Geometry
 import org.neo4j.graphdb.spatial.Point
+import org.neo4j.values.storable.CoordinateReferenceSystem
 import java.io.IOException
 import java.time.temporal.TemporalAccessor
 
-
-data class InternalPoint(val coordinates: List<Double>, val crs: CRS)
+data class StreamsPointCartesian(val crs: String, val x: Double, val y: Double, val z: Double?)
+data class StreamsPointWgs(val crs: String, val latitude: Double, val longitude: Double, val height: Double?)
 class PointSerializer : JsonSerializer<Point>() {
     @Throws(IOException::class, JsonProcessingException::class)
     override fun serialize(value: Point?, jgen: JsonGenerator,
                            provider: SerializerProvider) {
         if (value == null) {
             return
         }
-        val point = InternalPoint(value!!.coordinates[0].coordinate, value!!.crs)
+        val crsType = value!!.crs.type
```

Comment:
can we just add an extension method to InternalPoint?
So we can use it anywhere.